src/gentropy/assets/schemas/vep_json_output.json

```diff
@@ -20,6 +20,12 @@
         "containsNull": true,
         "elementType": {
           "fields": [
+            {
+              "metadata": {},
```

Comment:
No. Not exactly. The field `"hgvsg"` indeed repeated twice, but in the first instance it is within the `intergenic_consequences` object, the other one is in the `transcript_consequences` object. Unfortunately, because the VEP output is not perfectly modelled, we need both: when there's no transcript consequence we need to access this value from intergenic_conseqence and vice versa.